### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by dskonig-dev**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by dskonig-dev** [comment]
I changed the name in README.md file

**Review by dskonig-dev — COMMENTED** [review]
Test

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
   Add your open source license, GitHub uses Creative Commons Attribution 4.0 International.
 -->
 
-# Review pull requests
+# Review pull requests - JC
```
Comment (dev-dsk) [inline README.md:8]: add suggestion
```suggestion
# Review pull requests - JC
```

Comment (dskonig-dev) [inline README.md]:
```suggestion
# Review pull requests - JC Other Title
```

**Pull request merged**